src/main/app/src/app/shared/pipes/dagen.pipe.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {Pipe, PipeTransform} from '@angular/core';
+import * as moment from 'moment';
+import {TranslateService} from '@ngx-translate/core';
+
+@Pipe({name: 'dagen'})
+export class DagenPipe implements PipeTransform {
+
+    constructor(private translate: TranslateService) {}
+
+    transform(value: any): string {
+        if (value) {
+            const vandaag = moment(new Date()).startOf('day');
+            const vergelijkDatum = moment(value).startOf('day');
+            const aantalDagen = vergelijkDatum.diff(vandaag, 'days');
+
+            let result;
+            if (aantalDagen === 0) {
+                result = this.translate.instant('verloopt.vandaag');
+            } else if (aantalDagen <= -1) {
+                const dagen = vandaag.diff(vergelijkDatum, 'days');
```

Comment:
Vermeningvuldigen met -1 was ook wel genoeg geweest. Maar los daarvan, er is in de voorbespreking gezegd dat we niets zouden tonen als de datum al verlopen is. Dan staat immers het driehoekje er al. Ik weet niet of dat ook het geval is als hij vandaag verloopt, maar als het driehoekje er staat dan in deze kolom niets tonen.